content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,125 @@
+---
+Title: 'html_entity_decode()'
+Description: 'Converts HTML entities to their corresponding characters.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`html_entity_decode()`** function is the opposite of [`htmlentities()`](https://www.codecademy.com/resources/docs/php/string-functions/html-entities) function in that it converts HTML entities in the string to their corresponding characters.
+
+## Syntax
+
+```pseudo
+html_entity_decode(string,flags,encoding)
+```
+
+The `html_entity_decode()` function has one required parameter and three optional parameters:
+
+- `$string`: Required. Specifies the `string` to convert.
+- `$flags`: Optional. Specifies how to handle quotes, invalid encoding and the used document type. Ex: ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML5.
+- `$encoding`: An optional argument defining the encoding used when converting characters. If omitted, `$encoding` defaults to the value of the default_charset configuration option. Although this argument is technically optional, you are highly encouraged to specify the correct value for your code if the `default_charset` configuration option may be set incorrectly for the given input.
+
+
+  - The available quote styles are:
+
+    | Constant Name | Description|
+    | ------------- | ----------------------------------- |
+    | ENT_COMPAT    | Default. html_entity_decode() function encodes only double quotes from `$string`|
```

Comment:
```suggestion
    | `ENT_COMPAT`    | (Default) Encodes only double quotes from `$string`|
```

Reply:
Repeating the function here isn't really adding anything. Update the other lines as above